### PR TITLE
Classification of unknown package

### PR DESCRIPTION
**Pull request by crvreddy**

### CODE REVIEW

File: src/AritfactoryUploader.UTest/ArtifactoryUploaderTest.cs
[the remark this answers is not in the document]
Comment (crvreddy) [reply src/AritfactoryUploader.UTest/ArtifactoryUploaderTest.cs:1]: feature/SEPP_CLI_Incorporation changes are merged in this branch.No need to create new PR for this .

File: src/ArtifactoryUploader/Model/PackagesForErrorDisplay.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        public List<ComponentsToArtifactory> JfrogNotFoundPackagesMaven { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesNpm { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesNuget { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesConan { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesPython { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesDebian { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesMaven { get; set; }
+        public List<ComponentsToArtifactory> SuccessfullPackagesNpm { get; set; }
+        public List<ComponentsToArtifactory> SuccessfullPackagesNuget { get; set; }
+        public List<ComponentsToArtifactory> SuccessfullPackagesConan { get; set; }
```
Comment (Vijayalakshmi027) [inline src/ArtifactoryUploader/Model/PackagesForErrorDisplay.cs:40]: class name says packages for error display but members are named like successful. give proper name to the class

File: src/ArtifactoryUploader/PackageUploader.cs
Comment (Vijayalakshmi027) [inline src/ArtifactoryUploader/PackageUploader.cs:1]: have u  added UTs?

**Review by Vijayalakshmi027 — COMMENTED** [review]
add unit test wherever it is necessary and test cases and check it in the pipeline

File: src/ArtifactoryUploader/Model/PackagesForErrorDisplay.cs (hunk 1 above)
Comment (crvreddy) [reply src/ArtifactoryUploader/Model/PackagesForErrorDisplay.cs]: Updated class name

File: src/ArtifactoryUploader/PackageUploader.cs
Comment (crvreddy) [reply src/ArtifactoryUploader/PackageUploader.cs:1]: Its already there and I updated accordingly.

File: src/ArtifactoryUploader/PackageUploader.cs [hunk 2]
```diff
@@ -59,11 +67,69 @@ public static async Task UploadPackageToArtifactory(CommonAppSettings appSetting
...
             // set the error code
-            if(uploaderKpiData.PackagesNotUploadedDueToError > 0 || uploaderKpiData.PackagesNotExistingInRemoteCache > 0)
+            if (uploaderKpiData.PackagesNotUploadedDueToError > 0 || uploaderKpiData.PackagesNotExistingInRemoteCache > 0)
             {
                 Environment.ExitCode = 2;
                 Logger.Debug("Setting ExitCode to 2");
             }
         }
+
+        private static void DiplayAllSettings(List<Component> m_ComponentsInBOM, CommonAppSettings appSettings)
```
Comment (Vijayalakshmi027) [inline src/ArtifactoryUploader/PackageUploader.cs:77]: did u cover this method in UT

Comment (crvreddy) [reply src/ArtifactoryUploader/PackageUploader.cs:77]: yes its covered in PackageUploaderTest.cs

**Review by Vijayalakshmi027 — APPROVED** [review]

**Pull request merged**